tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -28,16 +28,12 @@ public function testDiffReturnsDiff()
         $diff = <<<'TXT'
 --- Original
 +++ New
-@@ @@
+@@ -1,3 +1,4 @@
  <?php
  class Foo extends Bar {
 -    function __construct($foo, $bar) {
 +    public function __construct($foo, $bar)
 +    {
-         $this->foo = $foo;
```

Comment:
btw. the old way of adding these trailing lines was why you committed https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/3204a0703f711ea791bd9b486e5d132c174d959e (which was reverted later on, but still ;))